### DISCUSSION TITLE
Ingenic T21 camera in home assistant

### DISCUSSION THREAD

**Discussion by Roberto00001** [community/community#142956] [Programming Help]
### Select Topic Area

Question

### Body

Hi,
I have seen that there are some users who tried to develop some code to integrate the generic SoC ingenic T21 (of many IP cameras) into Home Assistant, was anyone able to do it ?
thanks you !

**Comment by gtxaspec** [comment]
use a custom firmware, like thingino https://github.com/themactep/thingino-firmware, it's been used with home assistant before just fine, and works well with ingenic t21.
